src/services/navigationBar.ts

```diff
@@ -136,7 +136,7 @@ namespace ts.NavigationBar {
         for (let i = 0; i < depth; i++) endNode();
     }
     function startNestedNodes(targetNode: Node, entityName: EntityNameExpression) {
-        const names: Identifier[] = [];
+        const names: (Identifier | PrivateIdentifier)[] = [];

```

Comment:
Are our changes in navigationBar.ts correct? We added a passing test for navigation bar behavior, but couldn't figure out how to trigger this code path manually.